### ISSUE TITLE
Cannot import module '@nuxt/builder'

### ISSUE THREAD

**Issue by thely** [nuxt-community/svg-module#85]
I'm trying to figure out how to use this library, but every time I start up the dev server, I get the above error. I tried installing @nuxt/builder, but it didn't seem to impact anything. I have the package added to my buildModules, and I'm on nuxt v2.15.8.

**Comment by manuelodelain** [comment]
Hi @thely ! I have just tested it with a fresh install and I didn't see any issue. Can you provide a minimal example of your problem please?

**Comment by thely** [comment]
There's only one other person I've found online that had this particular error message, and I solved it the same way they did - I downloaded my project off GH into a new folder from its last working commit, reinstalled all modules, and then slowly added my changes back in.

The only thing I noticed that might have been the culprit was a rogue named component import, which I don't do anymore now that @nuxt/components is so reliable. Maybe Vetur threw one in without me realizing it when a component got moved around? Apart from that, I'm just as puzzled as the last person I saw with this issue. It's fixed, anyways, and this library (which is working just fine now!) clearly wasn't the source. :)
